src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Wrap help messages on form elements in `div` instead of `p`
+ * Add `form_label_content` and `form_help_content` block to form themes
```

Comment:
Can you please move this to a new 6.2 section?

Reply:
done